### PR TITLE
Implement eclipse/che Quarkus smoke test as prcheck

### PR DESCRIPTION
**Pull request by ScrewTSW**
### What does this PR do?
* Replaces original dashboard-happy-path with Quarkus smoke test as official PRCheck test
* Adds video recording and screenshots to artifacts

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/22045

### Is it tested? How?

IN-PROGRESS

#### Release Notes

Developed on Che Dogfooding instance as part of DevToolsWeek agenda
IN-PROGRESS

#### Docs PR

N/A

### CODE REVIEW

File: .ci/oci-dashboard-puppeteer.sh [hunk 1]
```diff
@@ -33,12 +33,11 @@ source <(curl -s ${DEVWORKSPACE_HAPPY_PATH}/common.sh)
...
   # Deploy Eclipse Che with a custom dashboard image
   cat >/tmp/che-cr-patch.yaml <<EOF
+apiVersion: org.eclipse.che/v2
 spec:
   components:
     dashboard:
-      deployment:
-        containers:
-          - image: ${CI_CHE_DASHBOARD_IMAGE}
+      dashboardImage: '${CI_CHE_DASHBOARD_IMAGE}'
```
Comment (olexii4) [inline .ci/oci-dashboard-puppeteer.sh:40]:
```suggestion
      deployment:
        containers:
          - image: ${CI_CHE_DASHBOARD_IMAGE}
             imagePullPolicy: Always
```

Comment (olexii4) [reply .ci/oci-dashboard-puppeteer.sh:40]: @ScrewTSW As I remember, ```dashboardImage: ...``` should refer to ```apiVersion: org.eclipse.che/v1```. 
WDYT?

**Comment by ibuziuk** [comment]
@olexii4 could you please take a look and follow up with @ScrewTSW on why the docker builds are failing?

File: .ci/oci-dashboard-puppeteer.sh (hunk 1 above)
Comment (ScrewTSW) [reply .ci/oci-dashboard-puppeteer.sh:40]: Good point. let me fix that in the follow-up PR

**Comment by ScrewTSW** [comment]
I'll be closing this PR in favor of the follow-up one.

**Pull request closed without merging**